calculator.test.js

```diff
@@ -6,16 +6,26 @@ var calculator = require('./calculator');
 
 describe('Calculator', function() {
   
-  it('should add two numbers', function() {
-    assert.equal(calculator.add(1,1), 2);
+  it('should add numbers', function() {
+    assert.equal(calculator.add(2,2,2,2,2), 10); // test will fail
```

Comment:
How about test case for add 2 numbers, does this function work well with add 2 numbers ?